<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>802</width>
    <height>840</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QFrame" name="frame_2">
      <property name="minimumSize">
       <size>
        <width>400</width>
        <height>500</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="PlotWidget" name="ChartWidget_1" native="true"/>
       </item>
       <item>
        <widget class="PlotWidget" name="ChartWidget_2" native="true"/>
       </item>
       <item>
        <widget class="PlotWidget" name="ChartWidget_3" native="true"/>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame">
      <property name="minimumSize">
       <size>
        <width>251</width>
        <height>450</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>251</width>
        <height>1500</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="upperLabel_1">
         <property name="text">
          <string>Choose testing method</string>
         </property>
         <property name="textFormat">
          <enum>Qt::AutoText</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="upperLabel_2">
         <property name="text">
          <string>and press 'Connect' to start</string>
         </property>
         <property name="textFormat">
          <enum>Qt::AutoText</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>Test Controls</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QPushButton" name="buttonStart">
            <property name="text">
             <string>Start</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="buttonStop">
            <property name="text">
             <string>Stop</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="buttonConnect">
            <property name="text">
             <string>Connect</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="buttonCamera">
            <property name="text">
             <string>Camera</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QPushButton" name="buttonAutoload">
            <property name="text">
             <string>Autoload</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QPushButton" name="buttonCalcLoad">
            <property name="text">
             <string>Calc Load</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QTabWidget" name="tabWidget">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tabLoad">
          <attribute name="title">
           <string>Load</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="2" column="0" colspan="2">
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>One direction time</string>
             </property>
            </widget>
           </item>
           <item row="1" column="4">
            <widget class="QLabel" name="label_18">
             <property name="text">
              <string>N</string>
             </property>
            </widget>
           </item>
           <item row="6" column="3">
            <widget class="QLineEdit" name="factorCyclNum">
             <property name="text">
              <string>5</string>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QLabel" name="label_17">
             <property name="text">
              <string>N</string>
             </property>
            </widget>
           </item>
           <item row="7" column="1">
            <widget class="QDoubleSpinBox" name="factorKp1">
             <property name="maximum">
              <double>1.000000000000000</double>
             </property>
             <property name="singleStep">
              <double>0.010000000000000</double>
             </property>
             <property name="value">
              <double>0.060000000000000</double>
             </property>
            </widget>
           </item>
           <item row="7" column="0">
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>Kp</string>
             </property>
            </widget>
           </item>
           <item row="2" column="3">
            <widget class="QLineEdit" name="factorTimeAx">
             <property name="text">
              <string>20</string>
             </property>
            </widget>
           </item>
           <item row="3" column="3">
            <widget class="QCheckBox" name="checkBoxCycl">
             <property name="text">
              <string>Cycled</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="label_11">
             <property name="text">
              <string>Axis 1</string>
             </property>
            </widget>
           </item>
           <item row="1" column="3">
            <widget class="QLineEdit" name="factorForceAx2">
             <property name="text">
              <string>2</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="factorForceAx1">
             <property name="text">
              <string>2</string>
             </property>
            </widget>
           </item>
           <item row="0" column="3">
            <widget class="QLabel" name="label_16">
             <property name="text">
              <string>Axis 2</string>
             </property>
            </widget>
           </item>
           <item row="6" column="0" colspan="2">
            <widget class="QLabel" name="label_15">
             <property name="text">
              <string>Number of Cycles</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Load</string>
             </property>
            </widget>
           </item>
           <item row="2" column="4">
            <widget class="QLabel" name="label_12">
             <property name="text">
              <string>s</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0" colspan="2">
            <widget class="QCheckBox" name="checkBoxFailure">
             <property name="text">
              <string>Until failure</string>
             </property>
            </widget>
           </item>
           <item row="8" column="0">
            <widget class="QLabel" name="label_13">
             <property name="text">
              <string>Kd</string>
             </property>
            </widget>
           </item>
           <item row="7" column="3">
            <widget class="QDoubleSpinBox" name="factorKp2">
             <property name="maximum">
              <double>1.000000000000000</double>
             </property>
             <property name="singleStep">
              <double>0.010000000000000</double>
             </property>
             <property name="value">
              <double>0.060000000000000</double>
             </property>
            </widget>
           </item>
           <item row="9" column="0">
            <widget class="QLabel" name="label_14">
             <property name="text">
              <string>Ki</string>
             </property>
            </widget>
           </item>
           <item row="8" column="1">
            <widget class="QDoubleSpinBox" name="factorKd1">
             <property name="decimals">
              <number>3</number>
             </property>
             <property name="maximum">
              <double>1.000000000000000</double>
             </property>
             <property name="singleStep">
              <double>0.001000000000000</double>
             </property>
            </widget>
           </item>
           <item row="9" column="1">
            <widget class="QDoubleSpinBox" name="factorKi1">
             <property name="decimals">
              <number>3</number>
             </property>
             <property name="maximum">
              <double>0.500000000000000</double>
             </property>
             <property name="singleStep">
              <double>0.001000000000000</double>
             </property>
             <property name="value">
              <double>0.002000000000000</double>
             </property>
            </widget>
           </item>
           <item row="8" column="3">
            <widget class="QDoubleSpinBox" name="factorKd2">
             <property name="decimals">
              <number>3</number>
             </property>
             <property name="maximum">
              <double>1.000000000000000</double>
             </property>
             <property name="singleStep">
              <double>0.001000000000000</double>
             </property>
            </widget>
           </item>
           <item row="9" column="3">
            <widget class="QDoubleSpinBox" name="factorKi2">
             <property name="decimals">
              <number>3</number>
             </property>
             <property name="maximum">
              <double>0.500000000000000</double>
             </property>
             <property name="singleStep">
              <double>0.001000000000000</double>
             </property>
             <property name="value">
              <double>0.002000000000000</double>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <spacer name="horizontalSpacer_4">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="0" column="2">
            <spacer name="horizontalSpacer_5">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="0" column="4">
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tabStrain">
          <attribute name="title">
           <string>Strain</string>
          </attribute>
         </widget>
         <widget class="QWidget" name="tabDisp">
          <attribute name="title">
           <string>Displacement</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="0" column="4">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Axis 2</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Speed</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Axis 1</string>
             </property>
            </widget>
           </item>
           <item row="3" column="2">
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="1" column="2">
            <widget class="QLineEdit" name="factorSpeedAx1">
             <property name="text">
              <string>3</string>
             </property>
            </widget>
           </item>
           <item row="1" column="5">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>mm/min</string>
             </property>
            </widget>
           </item>
           <item row="1" column="4">
            <widget class="QLineEdit" name="factorSpeedAx2">
             <property name="text">
              <string>3</string>
             </property>
            </widget>
           </item>
           <item row="1" column="3">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>mm/min</string>
             </property>
            </widget>
           </item>
           <item row="2" column="2">
            <widget class="QLineEdit" name="factorLength1">
             <property name="text">
              <string>3</string>
             </property>
            </widget>
           </item>
           <item row="2" column="4">
            <widget class="QLineEdit" name="factorLenght1_2">
             <property name="text">
              <string>3</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_19">
             <property name="text">
              <string>Length</string>
             </property>
            </widget>
           </item>
           <item row="2" column="3">
            <widget class="QLabel" name="label_20">
             <property name="text">
              <string>mm</string>
             </property>
            </widget>
           </item>
           <item row="2" column="5">
            <widget class="QLabel" name="label_21">
             <property name="text">
              <string>mm</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="title">
          <string>Motor Controls</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="3" column="0">
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>Axis 2</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Axis 1</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="buttonSampleP">
            <property name="text">
             <string>Move Sam. Pos.</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QPushButton" name="buttonMoveCentAx1">
            <property name="text">
             <string>Move Center</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QPushButton" name="buttonInitMot">
            <property name="text">
             <string>Init motors</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QPushButton" name="buttonMoveBackAx1">
            <property name="text">
             <string>Move Back</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QPushButton" name="buttonMoveCentAx2">
            <property name="text">
             <string>Move Center</string>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QPushButton" name="buttonMoveBackAx2">
            <property name="text">
             <string>Move Back</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="buttonZeroPosition">
            <property name="text">
             <string>Set Zero Position</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QPushButton" name="buttonZeroForce">
            <property name="text">
             <string>Set Zero Force</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="labelFolder">
           <property name="text">
            <string>Folder address</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="buttonFolder">
           <property name="maximumSize">
            <size>
             <width>20</width>
             <height>23</height>
            </size>
           </property>
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>802</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QWidget</extends>
   <header location="global">pyqtgraph</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
